designer/submodules/packages/components/src/common/nuIcon.tsx

```diff
@@ -1,8 +1,20 @@
 import React from "react";
 import loadable from "@loadable/component";
 import { styled } from "@mui/material";
+import { CrackedNut } from "./CrackedNut";
 
-const Icon = loadable(async () => import("nussknackerUi/Icon"));
+// This retry mechanism was introduced as sometimes dynamic import would fail.
+const Icon = loadable(() =>
+    import("nussknackerUi/Icon").catch(async (error) => {
+        console.warn("Failed to import Icon submodule. Import retry will be carried out", error);
+        try {
+            return await import("nussknackerUi/Icon");
```

Comment:
Only once? :( At least 3?

Reply:
This retry mechanism propably isn't a final solution as we do not fully understand why the promise sometimes rejects and whether one or even three retries would help with it.
With this approach we want to see whether a retry would even help and also gather some more information with the console logs.